doc/classes/RigidBody2D.xml

```diff
@@ -113,7 +113,7 @@
 			<return type="float">
 			</return>
 			<description>
-				Return the body's moment of inertia. This is usually automatically computed from the mass and the shapes. Note that this doesn't seem to work in a [code]_ready[/code] function: it apparently has not been auto-computed yet.
+				Return the body's moment of inertia. Automatically computes inertia from associated[class CollisionShape]s' mass during physics frames, after the [code]_ready[/code] function has triggered.
```

Comment:
- Return -> Returns
- Missing a space after "associated"
- Class links are [CollisionShape2D] - and since this is RigidBody2D, you should link to the 2D version.
- What does "_ready function has triggered" mean? I think you're trying to refer to the fact that it isn't computed until the next frame after the node is added to the tree, but this is not clear.

Overall, I don't think this definition is good. "from associated[class CollisionShape]s' mass" - a  CollisionShape2D doesn't have a mass. Moment of inertia is calculated from mass and the assigned shapes, which define the body's extents.